components/Navbar.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import Link from "next/link"
+import styles from '../styles/Nav.module.css'
+
+const Navbar = () => {
+  return (
+    <ul className={styles.ul}>
+      <li className={styles.li}>
+        <Link href="/">
+          <a>Home</a>
+        </Link>
+      </li>
+      <li className={styles.li}>
+        <Link href="/register">
+          <a>Register</a>
+        </Link>
+      </li>
+      <li className={styles.li}>
+        <Link href="/login">
+          <a>Login</a>
+        </Link>
+      </li>

```

Comment:
For stuff like this, I think we could create an array of objects that could house the data for the navbar. Maybe something like `navbarItems` and then we could:
```suggestion
      {navBarItems.map((item) => {
        return (
          <li className={styles.li}>
            <Link href={item.path}>
              <a>{item.name}</a>
            </Link>
          </li>
        )
      })}
```
I also saw this in your Sidebar component. This could also be applied there.